### ISSUE TITLE
pf::util::valid_mac miss some valid mac address

### ISSUE THREAD

**Issue by jrouzierinverse** [inverse-inc/packetfence#6419]
The commit fe136435 added an additional check to reject MAC addresses that look like IP addresses.

The following is a valid mac address WWW.XXX.YYY.ZZZ format.
With this formatting some MAC address can look like IP address when all the digits are numeric and each section is less than 256.

The is question are we going to support this type of MAC formatting.

Outside of switch lookup where do we have a scenario where a MAC and IP address can be confused for one another?

**Comment by julsemaan** [comment]
I don't believe there are vendors that use the format WWW.XXX.YYY.ZZZ for the MAC address

The closest is Cisco with XXXX.YYYY.ZZZZ but that doesn't match the IP pattern

IMO, I think this is an acceptable behavior, using that format for MAC addresses is just potential trouble so I don't see a vendor dumb enough to use it (although network vendors always have a way of making me lie for these)
